<launch>
    <node name="serial" pkg="rosserial_python" type="serial_node.py">
        <param name="port" value="/dev/hackerboard"/>
    </node>
    
    <node name="kalman_filter"  pkg="robot_exploration" type="odom_filter.py"/>
    <node name="dead_reckoning" pkg="robot_exploration" type="dead_reckoning" />

</launch>
